src/Symfony/Component/Messenger/EnvelopeItem.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger;
+
+/**
+ * An envelope item related to a message.
+ * This item must be serializable for transport.
+ *
+ * @author Maxime Steinhausser <maxime.steinhausser@gmail.com>
+ *
+ * @experimental in 4.1
+ */
+interface EnvelopeItem extends \Serializable
```

Comment:
By convention shouldn't it be named `EnvelopeItemInterface`

Reply:
200%!!! Let's be careful we should be consistent.

Reply:
Oops. Absolutely. It wasn't on purpose.